README.md

```diff
@@ -56,6 +56,9 @@ To register a device, run as root from LmP device:
  $ DEVICE_API=http://<IP of docker-compose host>/sign lmp-device-register [-T <TOKEN>] [-f <factory>]
 ~~~
 
+**NOTE** If you're emulating the device with QEMU and using its user-mode networking, the default guest IP is `10.0.2.2`.
+```
+
```

Comment:
It might be in the 10.0.2.0/24 space, but mine is for example 10.0.2.15. Just like docker is in 172.16.0.0/12 space.